### PR TITLE
Fix 1843 1844

### PR DESCRIPTION
**Pull request by maticzav**
---
title: Pull Request
description: Template to use in your Pull Requests.
---

Hey :wave:,

Thank you for submitting the PR. I am so excited to see what you've build. To shorten the merge time as much as possible, I recommend you check all the items on the list below before submitting it.

> PS.: No worries if some of them are missing, I will try to look at it ASAP :slightly_smiling_face:

**Things to look after:**

- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests and linter rules pass

**Description of changes:**

### CODE REVIEW

**Comment by znarf** [comment]
Looks good, but there is a merge conflict to fix.

Also it's containing changes from the other PR, which makes it more difficult to review (and merge in isolation).

**Comment by maticzav** [comment]
@znarf I'll wait for the first PR to get merged and fix the changes in this one afterwards

File: src/index.ts [hunk 1]
```diff
@@ -46,8 +46,10 @@ export function main(port: number): Server {
     require('probot/lib/apps/sentry'),
     require('probot/lib/apps/stats'),
   ]
-
-  process.on('unhandledRejection', probot.errorHandler)
+  ;(process as NodeJS.EventEmitter).on(
```
Comment (znarf) [inline src/index.ts:49]: Evil! ;-) Can that be avoided with a different syntax?

The reason why I prefer having semi-columns.

Comment (maticzav) [reply src/index.ts:49]: It's super ugly, I agree, but sadly that's the only way right now. We should remove it once the TS team resolves it.

**Pull request merged**